src/lib/compile-template.js

```diff
@@ -0,0 +1,28 @@
+import path from 'path';
+import fs from 'fs';
+import Handlebars from 'handlebars';
+let COMPILED_TEMPLATES = {};
+/**
+ * open template file sync (ONCE) and compile it!
+ * @param {String} templateName - filename of template
+ * @param {String} type - the file type
+ * @returns {Object} - compiled templates
+ */
+function compileTemplate (templateName, type) {
+  const filename = `${templateName}.${type}`;
+  const filepath = path.resolve('./src/templates', filename);
+
+  const templateCached = COMPILED_TEMPLATES[`${templateName}.${type}`];
+
+  const template = fs.readFileSync(filepath, 'utf8');
```

Comment:
There's no error handling in the case that the file can't be read.  If, for some reason, there's no file, line 18 would break.

Also, have you done `readFileSync` instead of `readFile` for a reason?